src/ansible_navigator/configuration_subsystem/configurator.py

```diff
@@ -234,7 +234,17 @@ def _apply_cli_params(self) -> None:
                 self._messages.append(LogMessage(level=logging.INFO, message=message))
 
     def _post_process(self) -> None:
+        delayed = []
```

Comment:
Is the order important? From a prompt look I got an impression that this could be a set, no?

Reply:
Order could be important in theory. Right now the post processors are run in alphabetical order. This keeps that, but just lets you move some subset of them to be run (alphabetically, still) after the first ones.

If you need things "delayed" more than one time (e.g. both PP `a` and `b` are delayed already, but `b` needs to depend on `a` for some reason), then you will be at the mercy of how they PPs are sorted.